src/components/issue-list-item.component.js

```diff
@@ -56,16 +66,23 @@ export const IssueListItem = ({ type, issue, navigation, language }: Props) =>
       <ListItem
         containerStyle={styles.listItemContainer}
         title={issue.title}
-        subtitle={`#${issue.number} - ${moment(issue.created_at).fromNow()}`}
+        subtitle={`#${issue.number} opened ${moment(
```

Comment:
@chinesedfan All phrases must be moved to `locale` files.

And let's rename `opened` to `created`, because our icon shows an open or closed issue.

Reply:
As soon as you get to this @chinesedfan, we'll merge this in :)

Reply:
@lex111 @housseindjirdeh Done. I read so quickly that missed the first line about locale files. 🤣